components/admin_console/plugin_management/plugin_management.tsx

```diff
@@ -360,6 +360,23 @@ const PluginItem = ({
         );
     }
 
+    if (pluginStatus.id === 'com.mattermost.apps') {
+        activateButton = (<></>);
+        removeButton = (<></>);
```

Comment:
0/5 Can these be set to `null` instead? I think that's usually the convention used in the webapp and mobile repos when we want to render nothing for a given local variable.